netmiko/ruckus/ruckus_fastiron.py

```diff
@@ -50,7 +57,7 @@ def enable(
                     if not re.search(
                         r"error.*incorrect.*password", new_data, flags=re.I
                     ):
-                        return output
+                        break
```

Comment:
@ktbyers Is it OK to break here and return the `output` at the end of this method? Will the following check  `if not self.check_enable_mode():` be false in case of correct output?

Reply:
Yes, that is fine. You will need to add that final return statement though (just making sure we are on the same page here). Yes, that final exception shouldn't happen (i.e. that means something unexpected went wrong...for example, the while loop completed with never seeing the username/password.

Reply:
Ohh, I see the `return output` now...it was just really hard to see in the GitHub diff.

Reply:
Should I change something here? :) This final `return output` should do the job, right?

Reply:
Yes, this should be good.